### ISSUE TITLE
Implement & document workflow for re-running failed job subsets & subsequent postprocessing.

### ISSUE THREAD

**Issue by rHorsey** [NREL/buildstockbatch#166]
*For a future date - if this is critical to you today please use the links below as a starting place to guide your efforts.*

**Is your feature request related to a problem? Please describe.**
When we're doing VERY large runs (i.e. 25mil sims for a full upgrade publication dataset) we will inevitable get HPC hicups (this last time a node with a not-working `/tmp/scratch` drive caused ~100 jobs to not complete). This isn't a huge deal normally, and even here it's not the end of the world, but it'd be nice to have a clear process to work through this sort of situation.

**Describe the solution you'd like**
Ensure job submission commands documented in sampling.out, include documentation on how to rerun a set of job IDs based off of this; figure out how to deal with associated post-processing re-running, since the `*.json.tar` files are deleted on postprocessing completion.

**Describe alternatives you've considered**
Not caring - generally a fine plan statistically speaking, but not so great for very public datasets. Lots of options in the postprocessing including lazy updates - yet to be explored. We could also try automated failure detection and resubmission but that sounds needlessly complex for current use cases.

**Additional context**

Relevant teams chat:

[6/26 3:29 PM] Merket, Noel
    You could call sbatch like [so](https://github.com/NREL/buildstockbatch/blob/master/buildstockbatch/eagle.py#L450-L459), but just include the job numbers you want to rerun in the --array parameter.
​[6/26 3:29 PM] Merket, Noel
    [https://slurm.schedmd.com/job_array.html](https://slurm.schedmd.com/job_array.html)
​[6/26 3:30 PM] Merket, Noel
    PROJECTFILE = the path to your project yaml file
​[6/26 3:30 PM] Merket, Noel
    MY_CONDA_ENV = that path to your conda environment
​[6/26 3:32 PM] Merket, Noel
    you can omit MEASURESONLY
​[10:10 AM] Horsey, Henry
    Sweet - many thanks! I did some more splunking around Friday and realized that once the postprocessing is complete it deletes the json.tar files, which makes it more complex to re-run failed jobs and then rebuild the metadata. I'm going to dive more into this after ComStock and IDSM are shipped - I think this is not too terribly important outside of huge runs, but in those higher publicity cases it'd be heart-warming to know that intermittent hardware failures in HPC land were easily rectifiable...
​[10:10 AM] Merket, Noel
    Ugh. Right.
​[10:11 AM] Horsey, Henry
    This'll probably end up being a PR from me at a later date... I'll make an issue for myself with this info and plan to work it later this year...
​[10:11 AM] Merket, Noel
    It's good to get rid of those intermediate files because they're redundant, but it does cause problems.

**Comment by afontani** [comment]
Closed with https://github.com/NREL/buildstockbatch/pull/304
